.swiftformat

```diff
@@ -0,0 +1,13 @@
+# file options
```

Comment:
Would it make sense to add this in a separate PR, since it's unrelated to the package sets?  Not a big thing but might make history more clear.